### PR TITLE
Also run CI on tag

### PR DESCRIPTION
**Pull request by lucacasonato**

### CODE REVIEW

**Review by bartlomieju — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ name: ci
 on: 
   push:
     branches: [main]
+    tags: ["**"]
```
Comment (ry) [inline .github/workflows/ci.yml:6]: I assume you know what you’re doing here. Two stars not one?

**Comment by piscisaureus** [comment]
why?

**Comment by lucacasonato** [comment]
@piscisaureus So that the CI can publish to crates.io on tag

**Comment by piscisaureus** [comment]
That’s how it always has been done, any idea what changed?

**Comment by ry** [comment]
@piscisaureus It seems it was broken in https://github.com/denoland/rusty_v8/commit/65692e80eb8b460b7cd1a64d44d8e1a8179586e9

Bartek ran into this while trying to do the release - it doesn't upload the artifacts to gh releases.

**Pull request merged**